packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2849,6 +2851,30 @@ class RenderMouseRegion extends RenderProxyBox implements MouseTrackerAnnotation
     }
   }
 
+  @override
+  bool get validForMouseTracker => _validForMouseTracker;
+  bool _validForMouseTracker;
+  set validForMouseTracker(bool value) {
+    if (_validForMouseTracker != value) {
+      _validForMouseTracker = value;
```

Comment:
Doing this probably requires `markNeedsBuild`. I would choose to not adding the setter until someone needs it (I'm not sure if one of the issues you're trying to fix does)

Reply:
(If we do keep this we probably want a test too)

Reply:
Done, nice suggestion :)